src/poggit/module/releases/index/PluginsByRepoReleaseListPage.php

```diff
@@ -42,22 +42,22 @@ public function __construct(string $query, array $filters) {
                 $semiTitlesBy[] = $pieces[0];
             } else {
                 assert(count($pieces) === 2);
-                $wheres[] = "(rp.owner = ? AND rp.name = ?";
+                $wheres[] = "(rp.owner = ? AND rp.name = ?)";
                 $type .= "ss";
                 $args[] = $pieces[0];
                 $args[] = $pieces[1];
                 $semiTitlesIn[] = "$pieces[0]/$pieces[1]";
             }
         }
-        $where = implode(" OR ", $wheres);
+        $where = "(" . implode(" OR ", $wheres) . ")";
         $plugins = Poggit::queryAndFetch("SELECT 
-            r.releaseId, r.name, r.version, rp.owner AS author, r.shortDesc,
+            r.releaseId AS releaseId, r.name AS name, r.version AS version, rp.owner AS author, r.shortDesc AS shortDesc,
```

Comment:
Indeed, that was leftover from my tests why the query was failing. Can't do any harm though!